Update textboxes on Keycloak login page and forgot password page to Material-UI style - WEB-583

jniweteto opened the pull request.

Commit:
add material ui to the login and forgot passowrd pages

Commit:
add material ui to the login and forgot passowrd pages - small adjustment

Commit:
small format adjust

Commit:
small format adjust

joshjabour reviewed (changes requested):
I'm requesting changes for 2 reasons:
1. The placeholder text "Username or Email" and "Password" is very small and needs to be bigger.
2. When you click on a field, highlighting appears in the wrong places. See the image below:
![image](https://user-images.githubusercontent.com/57398911/143867507-7bbdf93c-d13c-4be9-b3e0-a79665d06b39.png)

Feel free to remove the icons on the left side of the field. It should look like the demo here: https://material-components.github.io/material-components-web-catalog/#/component/text-field?type=outlined

Because this is Segun's design that we are trying to match: https://www.figma.com/file/RbZ3J9gognljk2KMGuoS8Y/A%2FB-Testing-of-Client-Reg?node-id=3095%3A0

Commit:
review correction

Commit:
review correction

Commit:
Squashed commit of the following:

commit 34950450a47c01abe3561b58e02c8373917100e0
Author: jniweteto <51609841+jniweteto@users.noreply.github.com>
Date:   Tue Nov 30 11:37:32 2021 +0200

    Update button color on Forgot Password page so it matches the OAF theme color (#2)

    Co-authored-by: jniweteto <ceteurcourageux@gmail.com>

commit a068426ad983b93771579482ebcab03ad4fa009f
Author: jniweteto <51609841+jniweteto@users.noreply.github.com>
Date:   Tue Nov 30 11:37:16 2021 +0200

    Add Kinyarwanda and language selector to keycloak theme (#1)

    * jsn-web-593 add Kinyarwanda to keycloak

    * added translations for forgot password

    * web-593 - add language selector to the login page

    * Update themes/oaf/login/login.ftl

    Co-authored-by: joshjabour <57398911+joshjabour@users.noreply.github.com>

    Co-authored-by: jniweteto <ceteurcourageux@gmail.com>
    Co-authored-by: joshjabour <57398911+joshjabour@users.noreply.github.com>

Commit:
Merge branch 'main' into jsn-web-583

Commit:
undo the changes in run.sh

joshjabour reviewed (changes requested):
Looks good, there's just one more change. I notice the top of the "P" in Password gets sliced off (not very noticeable, but hopefully you can see it in the screenshot). I don't know why it is getting sliced off, but it can be fixed by changing the font size to 14px for the placeholder text in all textboxes instead of 15px
![image](https://user-images.githubusercontent.com/57398911/145177160-fdc34be5-c9f0-4fa1-a503-9c048d702c01.png)

Commit:
changed the input font size in login forms

jniweteto commented:
> Looks good, there's just one more change. I notice the top of the "P" in Password gets sliced off (not very noticeable, but hopefully you can see it in the screenshot). I don't know why it is getting sliced off, but it can be fixed by changing the font size to 14px for the placeholder text in all textboxes instead of 15px ![image](https://user-images.githubusercontent.com/57398911/145177160-fdc34be5-c9f0-4fa1-a503-9c048d702c01.png)

Thanks for spotting this. I have changed the font size to 13px - 14px was still giving a sort of sliced P. I also adjusted the font sizes for the inputs so it becomes consistent.

Commit:
changed the input font size in login forms

joshjabour reviewed (approved):
Looks great 👍
